po/de.po

```diff
@@ -1440,9 +1528,9 @@ msgid ""
 "authentication data and has been disabled. To retry the login, re-enable the "
 "service."
 msgstr ""
-"Der Synchonisationsdienst <b>%s</b> kann sich nicht mit den angegeben  "
-"Authentifikations-Daten anmelden und wurde deaktiviert. Re-aktivieren Sie "
-"den Dienst, um die Anmeldung zu wiederholen."
+"Der Synchronisationsdienst <b>%s</b> kann sich nicht mit den angegeben  "
+"Authentifikation-Daten anmelden und wurde deaktiviert. Re-aktivieren Sie den "
```

Comment:
Maybe "Authentifikationsdaten" instead of "Authentifikation-Daten"? The latter sounds weird.